<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregar_usur_finalizar</class>
 <widget class="QDialog" name="agregar_usur_finalizar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asistente para agregar nuevo usuario</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QLabel" name="Lbl_imagenDisplay">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Una buena alineación de rostro, produce un mejor modelo de reconocimiento, para llevar dicha tarea es necesario ubicar los ojos. Utilizando el &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;ratón&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt; puedes modificar los campos de coordenadas de la derecha. Pulsa el &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;botón izquierdo&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt;, cuando encuentres el centro del ojo izquierdo. Homologamente haz lo mismo con el &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;botón derecho&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt; y el ojo derecho. El programa intenta ubicar las coordenadas automaticamente y son señalado mediante el centro de dos rectangulos.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>782</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Una buena alineación de rostro, produce un mejor modelo de reconocimiento. Pulsa el botón izquierdo, cuando encuentres el centro del ojo izquierdo. Homologamente haz lo mismo con el botón derecho y el ojo derecho. Modifica el offset, de modo que sólo aparezca en la imagen los ojos, nariz y boca. El software intenta ubicar los ojos automaticamente pero no simpre tiene buenos resultados, en tal caso pulsa recortar, para visualizar el resultado. Pulsa botón siguiente para continuar con la siguiente imagen.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>171</width>
     <height>90</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_ojoIzq">
        <property name="text">
         <string>Ojo izquierdo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LEdit_ojoIzq">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Coordenadas de ubicación de el ojo derecho.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_ojoDer">
        <property name="text">
         <string>Ojo derecho</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LEdit_ojoDer">
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Coordenadas de ubicación de el ojo izquierdo.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>% offset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="douSB_offset">
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Porcentaje de recorte, modificalo para  eliminar el fondo y que quede en lo posible, centrado el rostro.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="maximum">
         <double>0.400000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Btn_Recortar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;pre-visualiza el resultado, segun los volores proporcionados en los campos de arriba. Si el resultado es satisfactorio de click en el botón siguiente para continuar con la siguiente imagen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Recortar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Deshacer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Restaura a la imagen anterior, para volver a modificar las coordenadas.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Deshacer</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>540</y>
     <width>174</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_Siguiente">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Siguiente</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_Finalizar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Finalizar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>570</y>
     <width>801</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vl_StatusBar"/>
  </widget>
  <widget class="QLabel" name="Lbl_numImagen">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1/8</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
